app/models/review.rb

```diff
@@ -0,0 +1,27 @@
+# == Schema Information
+#
+# Table name: reviews
+#
+#  id            :integer          not null, primary key
+#  rating        :decimal(, )
+#  comment       :string
+#  created_at    :datetime         not null
+#  updated_at    :datetime         not null
+#  restaurant_id :integer
+#  user_id       :integer
+#
+
+class Review < ApplicationRecord
+  belongs_to :restaurant
+  belongs_to :user
+  validates :comment , :presence => true
+  validates :user_id, :restaurant_id, :presence => true, :allow_nil => false, :allow_blank => false
+
+
+
+  def update_restaurant_rating
+    cur_rest = Restaurant.find(self.restaurant_id)
```

Comment:
try to write full variable names to make the code more readable. In this case `current_restaurant`

Reply:
but in any case, in this code you could do:
put a method on Restaurant model called `update_rating!` like so:
```
def update_rating!
  update!(rating: calculate_rating)
end
```
and in here you would have:
``` 
def update_restaurant_rating!
  restaurant.update_rating!
end
```